### ISSUE TITLE
will add new feature that generate view's rest api?

### ISSUE THREAD

**Issue by Miazzy** [xgenecloud/xgenecloud#38]
**Feature request is in**
[ ] CLI
[ ] Desktop App
[ ] REST API Framework
[ ] GraphQL API Framework

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by RavishPatel-ICFNext** [comment]
[x] CLI
[x] Desktop App
[ ] REST API Framework
[x] GraphQL API Framework

Is your feature request related to a problem? Please describe.
For the MsSQL server, there are multiple Schemas and Servers that are linked. Creating a GraphQL object with links causes issues. It is easier to create a view by writing a query that fetches data from different schemas. Then we generate GraphQL APIs on top of the view. 

Describe the solution you'd like
"gen.apis.rest" would generate APIs for the Views as well.

Describe alternatives you've considered
NA

**Comment by o1lab** [comment]
Makes sense to write a view connecting to multiple sources. Views will be supported
